src/Nethermind/Nethermind.Serialization.Json/UInt256Converter.cs

```diff
@@ -29,8 +29,18 @@ private static UInt256 ReadInternal(ref Utf8JsonReader reader, JsonTokenType all
     {
         if (reader.TokenType == JsonTokenType.Number)
         {
-            return reader.GetUInt64();
+            if (reader.TryGetUInt64(out ulong smallValue))
+            {
+                return smallValue;
+            }
+
+            ReadOnlySpan<byte> bytes = reader.HasValueSequence ? reader.ValueSequence.ToArray() : reader.ValueSpan;
+            Span<char> chars = stackalloc char[bytes.Length + 1];
+            Encoding.UTF8.GetChars(bytes, chars);
```

Comment:
`Bytes.FromUtf8HexString` ?

Reply:
It's not hex, and it also outputs to a bytes span, but we need chars for `UInt256.Parse`